Source/SpatialGDK/Private/SpatialActorChannel.cpp

```diff
@@ -635,6 +657,12 @@ void USpatialActorChannel::OnCreateEntityResponse(const worker::CreateEntityResp
 		PinnedView->Remove(CreateEntityCallback);
 	}
 
+	// If a replicated stably named actor was created, update the GSM with the proper path and entity id
```

Comment:
// ... this ensures that each stably named actor is only created once.

Reply:
Added